apps/src/templates/manageStudents/Table/UsStateColumn/BulkSetModal/style.scss

```diff
@@ -0,0 +1,34 @@
+@import 'color';
+@import 'font';
+
+#us-state-column-bulk-set-modal {
+  font-weight: $medium-font-weight;
+
+  & #ui-close-dialog {
+    outline: none;
+  }
+
+  & a {
+    color: $default_text;
+    text-decoration: underline;
+  }
+
+  & hr {
+    border-color: $border_gray;
+    margin: 15px 0;
```

Comment:
Yes, we should use responsive units (`rem`/`em`) for responsive blocks, but only if other elements within the block also use these units. Otherwise, if a modal uses only `px`, mixing in responsive units may lead to inconsistencies across different pages